playwright/_impl/_network.py

```diff
@@ -83,6 +84,34 @@ def serialize_headers(headers: Dict[str, str]) -> HeadersArray:
     ]
 
 
+async def to_client_certificates_protocol(
+    clientCertificates: Optional[List[ClientCertificate]],
+) -> Optional[List[Dict[str, str]]]:
+    if not clientCertificates:
+        return None
+    out = []
+    for clientCertificate in clientCertificates:
+        out_record = {
+            "origin": clientCertificate["origin"],
+        }
+        if passphrase := clientCertificate.get("passphrase"):
```

Comment:
Note: walrus operator is supported since Python 3.8+ (last version we support) and helps us here esp. with optional types. since inside the if its e.g. not Optional anymore.